### PR TITLE
Issue #79: Improve wording on mutable default values.

### PR DESCRIPTION
**Pull request by ericvsmith**

### CODE REVIEW

File: pep-0557.rst [hunk 1]
```diff
@@ -426,19 +426,47 @@ field an initial value.
...
-      def __init__(self, x=[]):
+      x = []
+      def add(self, element):
+          self.x += element
+
+  o1 = C()
+  o2 = C()
+  o1.add(1)
+  o2.add(2)
+  assert o1.add == [1, 2]
```
Comment (ilevkivskyi) [inline pep-0557.rst]: This should be ``o1.x == [1, 2]``

File: pep-0557.rst [hunk 2]
```diff
@@ -426,19 +426,47 @@ field an initial value.
...
+      def add(self, element):
+          self.x += element
 
-  assert C().x is C().x
-  assert C().x is not C([]).x
+  assert D().x is D().x
 
-That is, two instances of class ``C`` that do not not specify a value
+This has the same issue as the original example using class ``C``.
+That is, two instances of class ``D`` that do not not specify a value
```
Comment (ilevkivskyi) [inline pep-0557.rst]: Duplicate word "do not not".

File: pep-0557.rst [hunk 3]
```diff
@@ -426,19 +426,47 @@ field an initial value.
...
+  o2 = C()
+  o1.add(1)
+  o2.add(2)
+  assert o1.add == [1, 2]
+  assert o1.x is o2.x
+
+Note that the two instances of class ``C`` share the same class
+variable ``x``, as expected.
+
+Using Data Classes, if this code was valid::
```
Comment (ilevkivskyi) [inline pep-0557.rst]: I would emphasize "if" using ``*if*`` reST formatting.

File: pep-0557.rst (hunk 1 above)
Comment (ericvsmith) [reply pep-0557.rst]: I thought I fixed that. I guess I fixed it in the REPL, but not the PEP code. Thanks.

File: pep-0557.rst (hunk 3 above)
Comment (ericvsmith) [reply pep-0557.rst]: Done.

File: pep-0557.rst (hunk 2 above)
Comment (ericvsmith) [reply pep-0557.rst]: Good catch. Thanks.

**Pull request merged**